ExeterTeam25/group-software-project
# Failed login

Test:
- pip install django-axes
- create a user
- login in with the username and wrong password 3 times
- should be locked out
- wait 5 minutes (can temporarily change to 1 minute for a faster test)
- login successfully
Details:
- added the django-axes module
- added functionality to lock the user out for 5 minutes if the user has 3 failed login attempt 
- updated readme
